Python/Product/Common/Strings.resx

```diff
@@ -2257,28 +2257,28 @@ Expand selection to the full expression?</value>
     <value>Debugger package could not be loaded</value>
   </data>
   <data name="PtvsdDisableCaption" xml:space="preserve">
-    <value>Disable the experimental debugger</value>
+    <value>Enable the legacy debugger</value>
   </data>
   <data name="PtvsdDisableSubtext" xml:space="preserve">
-    <value>Future debugging sessions will not use the experimental debugger. You can also change this setting in the Tools/Options/Python/Experimental page.</value>
+    <value>Future debugging sessions will use the legacy debugger. You can change this setting back in Tools &gt; Options &gt; Python &gt; Debugging page.</value>
```

Comment:
I normally use commas here and then break the list where the UI changes. e.g. "in Tools, Options, Python, Debugging and ensure that 'Use the legacy debugger' is checked" (and I carefully worded the "ensure ... is checked" part to avoid using control names)